### PR TITLE
support set options to create register, vote and return deposit coin tx

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: test/transaction/register_producer_tx.lua [hunk 1]
```diff
@@ -18,17 +18,80 @@ print(pubkey)
...
+
+if deposit_address == ""
+then
+    print("deposit addr is nil, should use --depositaddr or -daddr to set it.")
+    return
+end
+
+if own_publickey == ""
+then
+    print("owner public key is nil, should use --ownerpubkey or -opk to set it.")
```
Comment (peterwillcn) [inline test/transaction/register_producer_tx.lua]: should be --ownerpublickey

File: test/transaction/register_producer_tx.lua [hunk 2]
```diff
@@ -18,17 +18,80 @@ print(pubkey)
...
+    return
+end
+
+if node_publickey == ""
+then
+    print("node public key is nil, should use --nodepubkey or -npk to set it.")
```
Comment (peterwillcn) [inline test/transaction/register_producer_tx.lua]: should be --nodepublickey

File: cmd/script/script.go [hunk 3]
```diff
@@ -16,6 +16,94 @@ import (
 	"github.com/yuin/gopher-lua"
 )
 
+func registerParams(c *cli.Context, L *lua.LState) {
+	publicKey := c.String("pubkey")
```
Comment (peterwillcn) [inline cmd/script/script.go]: should be publickey

File: test/transaction/register_producer_tx.lua (hunk 1 above)
Comment (RainFallsSilent) [reply test/transaction/register_producer_tx.lua]: fixed

File: test/transaction/register_producer_tx.lua (hunk 2 above)
Comment (RainFallsSilent) [reply test/transaction/register_producer_tx.lua]: fixed

File: cmd/script/script.go (hunk 3 above)
Comment (RainFallsSilent) [reply cmd/script/script.go]: fixed

**Comment by peterwillcn** [comment]
ping @raindust

**Pull request merged**